javatests/arcs/core/data/expression/ExpressionTest.kt

```diff
@@ -154,6 +176,113 @@ class ExpressionTest {
         assertThat(evalExpression<Number, Number>(expr, currentScope, "arg" to 1)).isEqualTo(28)
     }
 
+    @Test
+    fun test_evalutor_paxel_from() {
+        val fromExpr = from<Number>("p") on "numbers"
+        assertThat(
+            evalExpression<Sequence<Number>, Sequence<Number>>(fromExpr, currentScope).toList()
+        ).isEqualTo(numbers)
+    }
+
+    @Test
+    fun test_evalutor_paxel_select() {
+        val selectExpr = from<Number>("p") on "numbers" select 1.asExpr()
+        assertThat(
+            evalExpression<Sequence<Number>, Sequence<Number>>(selectExpr, currentScope).toList()
+        ).isEqualTo(numbers.map { 1 })
+    }
+
+    @Test
+    fun test_evalutor_paxel_where() {
+        val whereExpr = from<Number>("p") on "numbers" where (currentScope["p"] eq 5.asExpr())
+        assertThat(
+            evalExpression<Sequence<Number>, Sequence<Number>>(whereExpr, currentScope).toList()
+        ).isEqualTo(listOf(5))
+    }
+
+    @Test
+    fun test_evalutor_paxel_max() {
+        val selectMaxExpr = from<Number>("p") on "numbers" select max(currentScope["numbers"])
+
+        assertThat(
+            evalExpression<Sequence<Number>, Sequence<Number>>(selectMaxExpr, currentScope).toList()
+        ).isEqualTo(numbers.map { numbers.size })
```

Comment:
you want `numbers.map { numbers.max() }` I think, it just so happens that max == size

Reply:
Done